src/types/location/location.ts

```
@@ -1,3 +1,15 @@
+import {
+  Bed,
+  Building,
+  Building2,
+  Car,
+  Eye,
+  Home,
+  Hospital,
+  LucideIcon,
+  Map,
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)